lightkurve/lightcurvefile.py

```diff
@@ -380,8 +380,16 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
                           LightkurveWarning)
 
         self.quality_bitmask = quality_bitmask
+        # Custom lightcurves may not have quality flags; assign zeros
+
+        if 'QUALITY' in self.hdu[1].data.columns.names:
+            quality_vector = self.hdu[1].data['QUALITY']
+        elif 'SAP_QUALITY' in self.hdu[1].data.columns.names:
+            quality_vector = self.hdu[1].data['SAP_QUALITY']
+        else:
+            quality_vector = self.hdu[1].data['TIME']*0.0
```

Comment:
I recommend using `np.zeros` here instead of a multiplication (because it'll be easier to read the code, and because this will break if TIME equals `nan` or `inf`).

Reply:
Also note: we may want to move this code block to a `_get_quality()` method in the base class, so that `KeplerLightCurveFile` can call it and behave in exactly the same way.